src/components/button.js

```diff
@@ -1,14 +1,15 @@
 import PropTypes from 'prop-types';
 
-const Button = ({ name, property }) => (
-  <button type="button" className={property}>
+const Button = ({ name, property, clickHandler }) => (
+  <button type="button" className={property} onClick={() => clickHandler(name)}>
     {name}
   </button>
 );
 
 Button.propTypes = {
   name: PropTypes.string,
   property: PropTypes.string,
+  clickHandler: PropTypes.func.isRequired,
```

Comment:
- [ ] Optional - Consider changing the component filename from `button.js` to `Button.js`. It's a convention to capitalize components filenames.